### PR TITLE
Adding install documentation for chef actions in standalone and combined

### PR DESCRIPTION
**Pull request by PrajaktaPurohit**
install mode

### CODE REVIEW

File: chef_master/source/install_actions.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+.. include:: ../../includes_actions/includes_actions.rst
+
+Requirements
+=====================================================
+.. include:: ../../includes_system_requirements/includes_system_requirements_actions.rst
+
+Install the Server
+=====================================================
+The |action| server can be installed as part of a standalone or high availability configuration.
```
Comment (PrajaktaPurohit) [inline chef_master/source/install_actions.rst]: rename to chef actions

File: chef_master/source/install_actions.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+-----------------------------------------------------
+.. include:: ../../includes_install/includes_install_actions_server_combined.rst
+
+Install the Client
+=====================================================
+.. include:: ../../includes_install/includes_install_actions_client.rst
+
+Upgrade the Server
+=====================================================
+The |chef actions| add-on can be upgraded as part of a standalone or high availability configuration:
```
Comment (PrajaktaPurohit) [inline chef_master/source/install_actions.rst]: there is no ha mode

File: includes_install/includes_install_actions_client.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+confirming the data has been sent.
+
+
+.. code-block:: bash
+
+   $ chef-client -l debug
+   ...
+   ...
+
+This includes the X-REMOTE-REQUEST-ID header of the chef server interactions in the run, which can be used in the
```
Comment (PrajaktaPurohit) [inline includes_install/includes_install_actions_client.rst:15]: do we want to mention knife plugin before it is created?

Comment (jamesc) [inline includes_install/includes_install_actions_client.rst:15]: actually there is no client piece at all - we log all actions on the server regardless of the client version.

The only difference is that the header is in 11.12 or later to correlate API calls from a single run or knife command

**Comment by jamesc** [comment]
Closing.  A version of this has already been merged

**Pull request closed without merging**